<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ballot1</class>
 <widget class="QDialog" name="Ballot1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Badea Catalina</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Florea Sergiu</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBox_4">
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tabaroiu Anca</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Manolache Andrei Laurentiu</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Buletin de vot: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Presedintele ascociatiei de studenti</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
